src/mumble/WASAPI.cpp

```diff
@@ -1065,7 +1065,9 @@ void WASAPIOutput::run() {
 	if (pwfx)
 		CoTaskMemFree(pwfx);
 
-	setVolumes(pDevice, false);
+	if (pDevice) {
+		setVolumes(pDevice, false);
```

Comment:
It’s only being called in `WASAPIOutput::run()` (3 times), after the initial pDevice null check which jumps to cleanup.